tests/unit/experimental/pipeline/test_pipeline.py

```diff
@@ -379,39 +379,48 @@ async def test_pipeline_async() -> None:
     assert pipeline_result[1].result == {"add": {"result": 12}}
 
 
-def test_pipeline_to_pgv() -> None:
+def test_pipeline_to_neo4j_viz() -> None:
     pipe = Pipeline()
     component_a = ComponentAdd()
     component_b = ComponentMultiply()
     pipe.add_component(component_a, "a")
     pipe.add_component(component_b, "b")
     pipe.connect("a", "b", {"number1": "a.result"})
-    g = pipe.get_pygraphviz_graph()
-    # 3 nodes:
+    g = pipe.get_neo4j_viz_graph()
+    # 4 nodes:
     #   - 2 components 'a' and 'b'
-    #   - 1 output 'a.result'
-    assert len(g.nodes()) == 3
-    g = pipe.get_pygraphviz_graph(hide_unused_outputs=False)
+    #   - 2 outputs 'a.result' and 'b.result' (neo4j-viz implementation includes both)
+    assert len(g.nodes) == 4
```

Comment:
Related to my comment above about logic, the test should not need to be updated.